sdk/core/amqp-common/package.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "@azure/amqp-common",
+  "version": "1.0.0-preview.3",
+  "description": "Common library for amqp based azure sdks like @azure/event-hubs.",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./typings/src/index.d.ts",
+  "dependencies": {
+    "@azure/ms-rest-nodeauth": "^0.9.2",
+    "@types/is-buffer": "^2.0.0",
+    "async-lock": "^1.1.3",
+    "buffer": "^5.2.1",
+    "debug": "^3.1.0",
+    "events": "^3.0.0",
+    "is-buffer": "^2.0.3",
+    "jssha": "^2.3.1",
+    "process": "^0.11.10",
+    "stream-browserify": "^2.0.2",
+    "tslib": "^1.9.3",
+    "url": "^0.11.0",
+    "util": "^0.11.1"
+  },
+  "browser": {
+    "./dist/index.js": "./browser/index.js",
+    "./dist-esm/src/index.node.js": "./dist-esm/src/index.browser.js",
+    "buffer": "buffer",
+    "stream": "stream-browserify"
+  },
+  "peerDependencies": {
+    "rhea-promise": "^0.1.15"
+  },
+  "devDependencies": {
+    "@types/async-lock": "^1.1.0",
+    "@types/chai": "^4.1.6",
+    "@types/chai-as-promised": "^7.1.0",
+    "@types/debug": "^0.0.31",
+    "@types/dotenv": "^6.1.0",
+    "@types/jssha": "^2.0.0",
+    "@types/mocha": "^5.2.5",
+    "@types/node": "^8.0.0",
+    "@types/sinon": "^5.0.5",
+    "assert": "^1.4.1",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "cross-env": "^5.2.0",
+    "dotenv": "^7.0.0",
+    "karma": "^4.1.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-mocha": "^1.3.0",
+    "mocha": "^5.2.0",
+    "mocha-junit-reporter": "^1.18.0",
+    "nyc": "^14.0.0",
+    "prettier": "^1.16.4",
+    "rhea-promise": "^0.1.15",
+    "rimraf": "^2.6.2",
+    "rollup": "^1.0.0",
+    "rollup-plugin-commonjs": "^9.2.0",
+    "rollup-plugin-inject": "^2.2.0",
+    "rollup-plugin-json": "^3.1.0",
+    "rollup-plugin-multi-entry": "^2.1.0",
+    "rollup-plugin-node-globals": "^1.4.0",
+    "rollup-plugin-node-resolve": "^4.2.0",
+    "rollup-plugin-replace": "^2.1.0",
+    "rollup-plugin-shim": "^1.0.0",
+    "rollup-plugin-sourcemaps": "^0.4.2",
+    "rollup-plugin-uglify": "^6.0.0",
+    "sinon": "^7.1.0",
+    "ts-node": "^7.0.1",
+    "tslint": "^5.15.0",
+    "typescript": "^3.2.2",
+    "ws": "^6.2.1"
+  },
+  "files": [
+    "browser/",
+    "dist/",
+    "dist-esm/src/",
+    "src/",
+    "typings/src/"
+  ],
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
```

Comment:
If we have removed the package-lock.json, we should update this command too, right? What should the correct command be? @bsiegel

Reply:
I believe as of now, since we are still going to be executing `npm install`, that would still generate the package-lock.json file being used by the scripts here.

All of the scripts can be updated later once the move to rush is complete